cmake/config/linux-gcc-clang.txt

```diff
@@ -91,16 +91,6 @@ set (c_compiler_flags_common
 set (cxx_compiler_flags_common
     -Wno-reorder
 )
-if (WARNINGS_AS_ERRORS)
-    if (CMAKE_BUILD_TYPE STREQUAL "Debug" OR
-        CMAKE_BUILD_TYPE STREQUAL "Profile" OR
-        CMAKE_BUILD_TYPE STREQUAL "Release")
-        set (c_compiler_flags_common
-            ${c_compiler_flags_common}
-            -Werror                                     # treat Warnings As Errors
-        )
-    endif ()
-endif ()
 set (cxx_compiler_flags_common
     ${cxx_compiler_flags_common}
     -std=c++11                                          # enable C++11
```

Comment:
We can replace this by:

set (CMAKE_CXX_STANDARD 11)

in the top level cmakefile